File: screenrecorder/src/main/java/com/sunplus/screenrecorder/media/glutils/IDrawer2dES2.java
package com.sunplus.screenrecorder.media.glutils;

/**
 * Created by w.feng on 2018/10/11
 * Email: fengweisb@gmail.com
 */
public interface IDrawer2dES2 extends IDrawer2D{
  int glGetAttribLocation(final String name);
  int glGetUniformLocation(final String name);
  void glUseProgram();
}

File: screenrecorder/src/main/java/com/sunplus/screenrecorder/media/glutils/IDrawer2dES2.kt
package com.sunplus.screenrecorder.media.glutils

/**
 * Created by w.feng on 2018/10/11
 * Email: fengweisb@gmail.com
 */
interface IDrawer2dES2 : IDrawer2D {
  fun glGetAttribLocation(name: String?): Int
  fun glGetUniformLocation(name: String?): Int
  fun glUseProgram()
}